### ISSUE TITLE
Feature request: allow configuration in TOML file with new  TOML-native tox config

### ISSUE THREAD

**Issue by Secrus** [ymyzk/tox-gh-actions#198]
In recent versions `tox` got support for native TOML configuration. It would be nice to be able to configure `tox-gh-actions` in TOML too.

**Comment by abelcheung** [comment]
The bizarre thing is, it already works! But there's some trick: the `version: factor` part must be specified as a multiline string:

```toml
[tool.tox.gh-actions]
python = """
	3.8: py38
	3.9: py39
	3.10: py310
	3.11: py311
	3.12: py312
	"""
```

It takes advantage of fact that `tox-gh-actions` internally parses this part using string processing. @ymyzk may decide to revamp the syntax for TOML support. Or maybe not.

**Comment by Secrus** [comment]
Yeah, but that is basically the same as what tox had before, inline INI syntax. Native TOML support should be nicer I guess.

**Comment by abelcheung** [comment]
> Native TOML support should be nicer I guess.

Wish it could be too, but apparently not now. In particular, `tox` TOML config doesn't support conditioanl factor currently, and there is no replacement except going back to ini. Perhaps this issue can go hand in hand with #178 .

Anyway, it's already a bless I don't need to throw away `tox-gh-actions` right now while migrating to TOML.

**Comment by bittner** [comment]
For TOML support it might be necessary or at least helpful to change the [`gh-actions:env`](https://github.com/ymyzk/tox-gh-actions/blob/master/src/tox_gh_actions/plugin.py#L95) key to `gh-actions.env` to align with Tox' [native TOML syntax](https://tox.wiki/en/stable/config.html#pyproject-toml-native).
